backend/env/.env.development

```diff
@@ -1,2 +1,4 @@
 NODE_ENV=development
-PORT=8000
\ No newline at end of file
+PORT=8000
+EXECUTOR_REGION=us-east-1
```

Comment:
Is there a reason we are using `us-east-1` instead of `ap-southeast-1`?

Reply:
No reason, it is the default region chosen by Serverless. Updated to deploy to ap-southeast-1 (Singapore) instead.